### ISSUE TITLE
A11y_Excel_File_ Share _ Screen reader: On Navigating in share dialog some of the controls such as "People in Microsoft with link can edit", "Message edit area", "Send button", "Edit copy link button" are not announced by NVDA.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#15125]
### Steps to reproduce:

1. Open Excel application.
2. Tab to file and press enter.
3. Tab to share and press enter.
4. "Send link" dialog box will appear.
5. Check whether the "People in Microsoft with link can edit", "Message edit area", "Send button", "Edit copy link button" are 
announcing by NVDA or not.

### Actual behavior:
On Navigating in share dialog "People in Microsoft with link can edit", "Message edit area", "Send button", "Edit copy link button" are not announced by NVDA.

### Expected behavior:
On Navigating in share dialog "People in Microsoft with link can edit", "Message edit area", "Send button", "Edit copy link button" should announce by NVDA.

### NVDA logs, crash dumps and other attachments:
https://github.com/nvaccess/nvda/assets/93735775/7207c30f-c642-40ab-9b58-1a3f38ffcadf

**Comment by msftedad** [comment]
Hi @seanbudd , Could you please provide update on this issue. Thanks.

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by seanbudd** [comment]
Hi, no updates

**Comment by Adriani90** [comment]
Testing with Microsoft 365 MSO (Version 2510 Build 16.0.19328.20178) 64 Bit
and NVDA alpha-53273,a6a536eb (2026.1.0.53273)

This issue does not occur anymore. NVDA imediately switches to browse mode in the share window, and I can also navigate the window in focus mode.
